cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateNamespaceCommand.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.cli.command;
+
+import co.cask.cdap.cli.ArgumentName;
+import co.cask.cdap.cli.ElementType;
+import co.cask.cdap.client.NamespaceClient;
+import co.cask.cdap.proto.NamespaceMeta;
+import co.cask.common.cli.Arguments;
+import co.cask.common.cli.Command;
+import com.google.inject.Inject;
+
+import java.io.PrintStream;
+
+/**
+ * Creates a CDAP namespace.
```

Comment:
Creates a namespace. *\[you don't say "Delete a CDAP namespace" in the following class, right?]*